examples/rds.txt

```diff
@@ -0,0 +1,111 @@
+RDS MYSQL AUDIT LOGS:
+    {
+        "messageType": "DATA_MESSAGE",
+        "owner": "908047316593",
+        "logGroup": "/aws/rds/instance/mysql-db-for-logs/audit",
+        "logStream": "mysql-db-for-logs",
+        "subscriptionFilters": [
+            "mysql-audit-filter"
+        ],
+        "logEvents": [
+            {
+                "id": "36283723329239606779976941014300561991466727829296513024",
+                "timestamp": 1627018425000,
+                "message": "20210723 05:33:45,ip-10-1-3-230,rdsadmin,localhost,203,0,CONNECT,,,0,SOCKET"
+            },
+            {
+                "id": "36282733332258008410023818770931443951379879892065320960",
+                "timestamp": 1626974032000,
+                "message": "20210726 12:05:10,ip-10-1-3-230,db_user,157.25.19.100,1029,242734,QUERY,`testdb`,'select * from mysql.user\nLIMIT 0, 1000',1142,,"
+            },
+            {
+                "id": "36282735473129547468963640358360398125622584466185256962",
+                "timestamp": 1626974128000,
+                "message": "20210722 17:15:28,ip-10-1-3-230,admin,157.25.19.100,13,0,DISCONNECT,testdb,,0,SSL"
+            }
+        ]
+    }
+
+RDS MYSQL GENERAL LOGS:
+    {
+        "messageType": "DATA_MESSAGE",
+        "owner": "908047316593",
+        "logGroup": "/aws/rds/instance/mysql-db-for-logs/general",
+        "logStream": "mysql-db-for-logs",
+        "subscriptionFilters": [
+            "mysql-general-filter"
+        ],
+        "logEvents": [
+            {
+                "id": "36283739563646893425505853973335389931765810644969062400",
+                "timestamp": 1627019152976,
+                "message": "2021-07-23T05:45:52.976229Z\t  199 Query\tUSE testdb"
+            },
+            {
+                "id": "36283739579569625497256718896391892778436740760239079426",
+                "timestamp": 1627019153690,
+                "message": "2021-07-23T05:45:53.690141Z\t  198 Query\tSHOW FULL TABLES FROM `testdb`"
+            },
+            {
+                "id": "36283739597923138795647421741875788916826342279660961797",
+                "timestamp": 1627019154513,
+                "message": "2021-07-23T05:45:54.513995Z\t  199 Query\tSHOW FULL COLUMNS FROM `testdb`.`customers`"
+            }
+        ]
+    }
+
+RDS MYSQL SLOWQUERY LOGS:
+    {
+        "messageType": "DATA_MESSAGE",
+        "owner": "908047316593",
+        "logGroup": "/aws/rds/instance/mysql-db-for-logs/general",
```

Comment:
Like @wojtasskorcz mentioned [here](https://github.com/dynatrace-oss/dynatrace-aws-log-forwarder/pull/32#discussion_r678916731) isn't this some copy paste error?

![image](https://user-images.githubusercontent.com/50921490/127482870-59003a2c-cd7f-4e7b-be4f-b51eab0701ee.png)

Reply:
Yes, it's copy paste error...
![image](https://user-images.githubusercontent.com/72859155/127965623-b5e20e8d-a59a-4a5a-90b7-786bd6753f81.png)